src/StackExchange.Redis/Enums/ClientFlags.cs

```diff
@@ -59,5 +69,17 @@ public enum ClientFlags : long
         /// connection to be closed ASAP
         /// </summary>
         CloseASAP = 256,
+        /// <summary>
+        /// the client is a Pub/Sub subscriber
+        /// </summary>
+        PubSubSubscriber = 512,
+        /// <summary>
+        /// the client is in readonly mode against a cluster node
+        /// </summary>
+        ReadOnlyCluster = 512,
+        /// <summary>
+        /// the client is connected via a Unix domain socket
+        /// </summary>
+        UnixDomainSocket = 512,
```

Comment:
Are these intended additions in this PR? Doesn't seem related...and they're all 512